internal/manifests/var.go

```diff
@@ -28,6 +35,19 @@ const (
 	LabelQueryFrontendComponent string = "query-frontend"
 )
 
+const (
+	// labelJobNameDistributor is the label value for the distributor serviceMonitor job label
+	labelJobNameDistributor string = "monitor-loki-distributor"
+	// labelJobNameIngester is the label value for the ingester serviceMonitor job label
+	labelJobNameIngester string = "monitor-loki-ingester"
+	// labelJobNameQuerier is the label value for the querier serviceMonitor job label
+	labelJobNameQuerier string = "monitor-loki-querier"
+	// labelJobNameCompactor is the label value for the compactor serviceMonitor job label
+	labelJobNameCompactor string = "monitor-loki-compactor"
+	// labelJobNameQueryFrontend is the label value for the query-frontend serviceMonitor job label
+	labelJobNameQueryFrontend string = "monitor-loki-query-frontend"
+)
```

Comment:
These constants are used as values for the ServiceMonitorSpec's `JobLabel` field. According to the docs:
```
❯ oc explain servicemonitor.spec.jobLabel
KIND:     ServiceMonitor
VERSION:  monitoring.coreos.com/v1

FIELD:    jobLabel <string>

DESCRIPTION:
     The label to use to retrieve the job name from.
```

`JobLabel` refers to the metrics label that should be used to identify the scraping job. AFAICS, we expose only the following labels on the services:
```
"app.kubernetes.io/name":     "loki",
"app.kubernetes.io/provider": "openshift",
"loki.grafana.com/name":       stackName,
"loki.grafana.com/component": component,
```